src/main/kotlin/the/coding/force/exploring_kotlin_coroutines/service/coroutine/CreateDataServiceCoroutine.kt

```diff
@@ -0,0 +1,24 @@
+package the.coding.force.exploring_kotlin_coroutines.service.coroutine
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.withContext
+import mu.KotlinLogging
+import org.springframework.stereotype.Service
+import the.coding.force.exploring_kotlin_coroutines.dto.DataDto
+import the.coding.force.exploring_kotlin_coroutines.dto.toEntity
+import the.coding.force.exploring_kotlin_coroutines.repository.DataRepository
+
+@Service
+class CreateDataServiceCoroutine(
+    private val dataRepository: DataRepository,
+    private val ioDispatcher: CoroutineDispatcher
+) {
+    private val logger = KotlinLogging.logger { }
+
+    suspend fun create(dataDto: DataDto) {
+        withContext(ioDispatcher) {
+            dataRepository.save(dataDto.toEntity())
+            logger.info { "CreateDataService.create, entity saved" }
```

Comment:
se quiser colocar um log diferente por ser com coroutine
```suggestion
            logger.info { "CreateDataService.create with coroutine, entity saved" }
```

Reply:
Boa ideia, isso ajuda diferenciar qual serviço está sendo utilizado, se é o que usa coroutine ou se é o que usa sem coroutine

Reply:
commit: [improving logs, change Dispatcher strategy, remove webFlux dependency](https://github.com/the-coding-force/exploring-kotlin-coroutines__backend__spring-kotlin__poc/pull/13/commits/bcb1229dfa29acc3455c10a16adc4b18b4edd81d)